enabler/test/de/schildbach/pte/live/DsbProviderLiveTest.java

```diff
@@ -76,7 +76,7 @@ public void suggestLocations() throws Exception {
     public void shortTrip() throws Exception {
         final QueryTripsResult result = queryTrips(
                 new Location(LocationType.STATION, "900000011", null, "Copenhagen Airport"), null,
-                new Location(LocationType.STATION, "551922500", null, "Billund Airport"), new Date(), true, Product.ALL,
+                new Location(LocationType.POI, "551922500", null, "Billund Airport"), new Date(), true, Product.ALL,
```

Comment:
Why this change? Is the 'Billund Airport' station not existing any more?

Reply:
The test failed for me without this change. There is no Billund Airport (train)station, but there exists bus stops.